src/software/ai/hl/stp/play/kickoff_friendly_play.cpp

```diff
@@ -129,8 +129,7 @@ void KickoffFriendlyPlay::getNextTactics(TacticCoroutine::push_type &yield,
         // in the middle of the enemy field
         kickoff_chip_tactic->updateControlParams(
             world.ball().position(),
-            world.field().centerPoint() + Vector(world.field().xLength() / 6, 0),
-            world.field().xLength() / 2);
+            world.field().centerPoint() + Vector(world.field().xLength() / 6, 0));
```

Comment:
What's this change for?

Reply:
I've removed the chip_distance if we define the chip_target, since chip_distance = chip_target-chip_origin